test/visual/equal-width-tabs-lumo.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+  <meta charset="UTF-8">
+  <title>Equal width tabs visual tests</title>
+</head>
+
+<body>
+  <style media="screen">
+    .capture-block {
+      display: block;
+      width: 800px;
+    }
+  </style>
+
+  <div class="capture-block" id="equal-width-tabs">
+    <vaadin-tabs theme="equal-width-tabs">
+      <vaadin-tab>Tab one</vaadin-tab>
+      <vaadin-tab>Tab two with a longer title</vaadin-tab>
+      <vaadin-tab>Tab three</vaadin-tab>
+    </vaadin-tabs>
+    <br>
+    <div style="display: flex; justify-content: center; width: 800px;">
```

Comment:
I guess we are checking the tabs in flex container, because of:
```
:host([theme~="equal-width-tabs"]) {
    flex: 1 1 auto;
}
```
But why do we need two of those? Will it be enough to use only one in `capture-block` which can be flex.